Packs/Zoom/Integrations/Zoom/Zoom.py

```diff
@@ -1,148 +1,621 @@
+        raise DemistoException("One or more arguments that were filed are used for recurring meeting with fixed time only")
+
+    if num_type == 8 and recurrence_type != 3 and any((args.get("monthly_day"),
+                                                       monthly_week, monthly_week_day)):
+        raise DemistoException(
+            "One or more arguments that were filed are for recurring meeting with fixed time and monthly recurrence_type only")
+
+    if num_type == 8 and recurrence_type == 3 and not (monthly_week and monthly_week_day) and not args.get("monthly_day"):
+        raise DemistoException(
+            """Missing arguments. recurring meeting with fixed time and monthly recurrence_type
+            must have the fallowing arguments: monthly_week and monthly_week_day""")
+
+    if num_type == 8 and recurrence_type != 2 and args.get("weekly_days"):
+        raise DemistoException("Weekly_days is for weekly recurrence_type only")
+
+    if num_type == 8 and not recurrence_type:
+        raise DemistoException(
+            "Missing arguments. recurring meeting with fixed time is missing this argument: recurrence_type")
+
+    # converting separately after the argument checking, because 0 as an int is equaled to false
+    jbh_time = arg_to_number(jbh_time)
+
+    if start_time:
+        check_start_time_format(start_time)
+
+    json_all_data = {}
+
+    # special section for recurring meeting with fixed time
+    if num_type == 8:
+        if recurrence_type == "Daily":
+            recurrence_type = 1
+        elif recurrence_type == "Weekly":
+            recurrence_type = 2
+        elif recurrence_type == "Monthly":
+            recurrence_type = 3
+        json_all_data.update({"recurrence": {
+            "end_date_time": end_date_time,
+            "end_times": end_times,
+            "monthly_day": monthly_day,
+            "monthly_week": monthly_week,
+            "monthly_week_day": monthly_week_day,
+            "repeat_interval": repeat_interval,
+            "type": recurrence_type,
+            "weekly_days": weekly_days
+        }})
+    json_all_data.update({
+        "settings": {
+            "auto_recording": auto_record_meeting,
+            "encryption_type": encryption_type,
+            "host_video": host_video,
+            "jbh_time": jbh_time,
+            "join_before_host": join_before_host,
+            "meeting_authentication": meeting_authentication,
+            "waiting_room": waiting_room
+        },
+        "start_time": start_time,
+        "timezone": timezone,
+        "type": num_type,
+        "topic": topic,
+    })
+    # remove all keys with val of None
+    json_data = remove_None_values_from_dict(json_all_data)
+    url_suffix = f"users/{user_id}/meetings"
+    # call the API
+    raw_data = client.zoom_create_meeting(url_suffix=url_suffix, json_data=json_data)
+    # parsing the response
+    if type == "recurring meeting with fixed time":
```

Comment:
const please.

Reply:
not my code. will be replaced in the new PR